WordPress/WordPressTest/ContextManagerMock.m

```diff
@@ -79,40 +76,13 @@ - (NSManagedObjectContext *)mainContext
     return _mainContext;
 }
 
-- (void)saveContext:(NSManagedObjectContext *)context
-{
-    [self saveContext:context withCompletionBlock:^{
-        if (self.testExpectation) {
-            [self.testExpectation fulfill];
-            self.testExpectation = nil;
-        } else if (self.requiresTestExpectation) {
-            NSLog(@"No test expectation present for context save");
-        }
-    }];
-}
-
 - (void)saveContextAndWait:(NSManagedObjectContext *)context
 {
     [super saveContextAndWait:context];
-    if (self.testExpectation) {
-        [self.testExpectation fulfill];
-        self.testExpectation = nil;
-    } else if (self.requiresTestExpectation) {
-        NSLog(@"No test expectation present for context save");
-    }
-}
-
-- (void)saveContext:(NSManagedObjectContext *)context withCompletionBlock:(void (^)(void))completionBlock
-{
-    [super saveContext:context withCompletionBlock:^{
-        if (self.testExpectation) {
-            [self.testExpectation fulfill];
-            self.testExpectation = nil;
-        } else if (self.requiresTestExpectation) {
-            NSLog(@"No test expectation present for context save");
-        }
-        completionBlock();
-    }];
+    // FIXME: Remove this method to use superclass one instead
+    // This log magically resolves a deadlock in
+    // `ZDashboardCardTests.testShouldNotShowQuickStartIfDefaultSectionIsSiteMenu`
+    NSLog(@"Context save completed");
```

Comment:
I'm still trying to figure out cause of this deadlock 👀

Reply:
For what is worth, I can see the deadlock without this line on my end, too. The test you mention just sits there with a spinner next to it in the test navigator.